docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+
+services: 
+  label_studio:
+    container_name: label_studio_server
+    build: .
+    working_dir: /label-studio
+    command: "label-studio start my_project --init --force"
```

Comment:
Should we mount an external volume here?

Reply:
You can do this by the following:
```yaml
command: "label-studio start my_project --init --root-dir /data"
volumes:
   - ./label-studio-projects:/data
```

Reply:
Sorry for late response, lots of works related to corona these days...
New changes have been tested including readme file.
Waiting your feedback.
Thank you!

Reply:
@CrazyOptimist thanks for your contribution! hope the pandemic doesn't break your goals :)

Reply:
Many thanks @niklub.
And wish you the best all the contributors!